site/pages/create-disruption.page.tsx

```diff
@@ -97,86 +213,100 @@ const CreateDisruption = (initialState: PageState<Partial<DisruptionPageInputs>>
-                            disabled={pageState.inputs.disruptionNoEndDateTime === "true"}
+                            value={validity.disruptionEndTime}
+                            disabled={validity.disruptionNoEndDateTime === "true"}
                             inputName="disruptionEndTime"
-                            stateUpdater={stateUpdater}
+                            stateUpdater={validityStateUpdater}
                             initialErrors={pageState.errors}
-                            schema={createDisruptionSchema.shape.disruptionEndTime}
+                            reset={!validity.disruptionEndTime}
+                            schema={validitySchema.shape.disruptionEndTime}
                         />
 
-                        <Checkbox<DisruptionPageInputs>
+                        <Checkbox<Validity>
                             inputName="disruptionNoEndDateTime"
                             display="Does the disruption have an end datetime?"
                             hideLegend
                             checkboxDetail={[
                                 {
                                     display: "No end date/time",
                                     value: "true",
-                                    checked: pageState.inputs.disruptionNoEndDateTime === "true",
+                                    checked: validity.disruptionNoEndDateTime === "true",
                                 },
                             ]}
-                            stateUpdater={stateUpdater}
-                            initialErrors={pageState.errors}
-                        />
-
-                        <Radios<DisruptionPageInputs>
-                            display="Does this disruption repeat?"
-                            value={pageState.inputs.disruptionRepeats ?? ""}
-                            inputName="disruptionRepeats"
-                            radioDetail={[
-                                {
-                                    value: "yes",
-                                    display: "Yes",
-                                },
-                                {
-                                    value: "no",
-                                    display: "No",
-                                },
-                            ]}
-                            paddingTop={6}
-                            stateUpdater={stateUpdater}
+                            stateUpdater={validityStateUpdater}
                             initialErrors={pageState.errors}
+                            reset={!validity.disruptionNoEndDateTime}
+                            schema={validitySchema.shape.disruptionNoEndDateTime}
                         />
+                        <button
+                            className="govuk-button govuk-button--secondary mt-8"
+                            data-module="govuk-button"
+                            onClick={addValidity}
+                            disabled={
```

Comment:
If a validity period is added with start date of 20/04/2023 and no end date then the button is disabled. This means a user is unable to add another validity between current date and 20/04

Reply:
yes this is intended